### ISSUE TITLE
Add a wiki: Rainverse Wiki

### ISSUE THREAD

**Issue by AverageHelper** [KevinPayravi/indie-wiki-buddy#637]
Link to origin wiki: https://rain-web-comic.fandom.com, https://myimpossiblesoulmate.fandom.com
Link to destination (independent) wiki: https://rainverse.wiki

(EDITED to add second origin wiki. Both are part of the same larger universe by the same author. The new wiki covers both.)

**Comment by 1bcb** [comment]
The [announcement on fediverse that we forked](https://transfem.social/notes/9qsqdkmqi78e01bh) the wiki was liked and boosted by Rainverse Creator Jocelyn Samara.  I'm not sure if that qualifies us as an "official" wiki.

**Comment by DrWhoFan13** [comment]
> The [announcement on fediverse that we forked](https://transfem.social/notes/9qsqdkmqi78e01bh) the wiki was liked and boosted by Rainverse Creator Jocelyn Samara. I'm not sure if that qualifies us as an "official" wiki.

The extension doesn't care about legitimacy, it cares about being more informative than the Fandom wiki. Quoth the website:
- The wiki should be editable by the public (that's what makes a wiki a wiki!)
- The destination wiki should be independent, meaning that decision-making and control of the wiki largely rests with the wiki's staff and volunteer editing community.
-  Factors include editorial independence, the editorbase's ability to request changes from their host, and any history of the host exerting decisions contrary to a wiki community's wishes.
-  Large hosts such as Miraheze and Wiki.gg may be considered less independent, but are included as they are community-friendly hosts, provide decent levels of control, and are popular destinations for forking wikis.
- The destination wiki should be of decent quality, ideally matching (or exceeding) the quality and size of the origin wiki. While we want to support all independent wikis, we also want extension users to be directed to wikis where they can find what they are looking for.
-  Should be of general interest. Wikis about particularly niche subjects may be discluded.
- When there are multiple independent wikis on the same subject, we will usually point to the wiki that is most complete and prominent in the community.

TL;DR: If it's boosted by the creator, that's a bonus; not a necessity. We don't care about "official" wikis, we care about _informative_ wikis.

**Comment by KevinPayravi** [comment]
@DrWhoFan13 We do track which wikis are official. It's not a requirement to be included, but we mention when wikis are official in some places in the UI (e.g. the redirect banner).

**Comment by 1bcb** [comment]
Another thing to note is that our wiki covers content equivalent to _both_ [Rain Fandom Wiki](https://rain-web-comic.fandom.com/wiki/Rain_(web_comic)_Wikia) and [My Impossible Soulmate Fandom Wiki](https://myimpossiblesoulmate.fandom.com/wiki/My_Impossible_Soulmate_Wiki).

**Comment by DrWhoFan13** [comment]
> Another thing to note is that our wiki covers content equivalent to _both_ [Rain Fandom Wiki](https://rain-web-comic.fandom.com/wiki/Rain_(web_comic)_Wikia) and [My Impossible Soulmate Fandom Wiki](https://myimpossiblesoulmate.fandom.com/wiki/My_Impossible_Soulmate_Wiki).

So a two for one deal, then.

**Comment by KevinPayravi** [comment]
Thank you for the submission! Added in `v3.7.1` (now available for Chrome, release pending for Firefox).
